lib/capybara/webkit/matchers.rb

```diff
@@ -7,11 +7,11 @@ module RspecMatchers
           actual.error_messages.any?
         end
 
-        failure_message_for_should do |actual|
+        failure_message do |actual|
```

Comment:
Will `alias_method` actually work? We're *calling* `failure_message_for_should`, not defining it.